ContactManger/src/main/java/FillContactData.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class FillContactData {
+
+    List<Contact> contacts;
+    Random random;
+
+    public FillContactData() {
+        contacts = new ArrayList<>();
+        random = new Random();
+    }
+
+    public List<Contact> fillContactData(int cont) {
+        for (int i = 0; i < cont; i++) {
+            String name = randomName();
+            String email = name + "@gmail.com";
+            String phone = String.valueOf(randomNumeric());
+            contacts.add(new Contact(name, email, phone));
+        }
+        return contacts;
+    }
+
+    private String randomName() {
+        String[] chars = {"Maria", "Mauricio", "Pedro", "Juan", "Marcela"};
```

Comment:
@anamariaTapia The variable names should be something more accurate to the value of it, for example "names"